OpenFEC-Data-Visualization/blog.md

```diff
@@ -0,0 +1,285 @@
+# Introduction to OpenFEC Data Visualization with ChartJs
+## Purpose
+
+The importance of understanding politics and the implications of being informed voters are profound. The ability for citizens of a country to understand the financial spending of candidates running for office (at any level) allows one to understand where a candidate's priorities lie. Observing and analyzing financial trends allows one to make predictions about who may win a particular election. A lot of the time, however, it’s easy to get lost in the numbers! 
+
+Thus, being able to visualize this data in a graphical manner helps in truly comparing and understanding a candidate’s financial spending in comparison to that of others in an easy manner. However, how do you verify the graphs you see online are indeed accurate? 
```

Comment:
To make the first sentence ("Thus...") more concise, I recommend phrasing as "Thus, being able to visualize this data helps in comparing and understanding the financial spending of candidates"